extensions/amp-analytics/0.1/events.js

```diff
@@ -661,15 +676,41 @@ class TimerEventHandler {
   /**
    * @param {!Window} win
    */
-  clearInterval(win) {
+  stopTimer_(win) {
```

Comment:
do we trigger when stopping the timer?

Reply:
Yes. Otherwise we would lose time in reporting.